### PR TITLE
Sample_methods, sample, contracts and quotation index

### PR DESCRIPTION
**Pull request by louMon**

### CODE REVIEW

File: app/views/contracts/_contracts_table.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<div class="content">
+        <div class="material-datatables">
+          <table id="contract_table" class="table table-no-bordered table-hover" >
```
Comment (Orange-walker-95) [inline app/views/contracts/_contracts_table.html.erb]: Use 4 spaces instead of 2 tabs or 2 spaces for indentation

Comment (Orange-walker-95) [inline app/views/contracts/_contracts_table.html.erb]: Use a '-' instead of a '_' for the ids names.

File: app/views/contracts/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<h1>
+	<div class="row">
+		<div class="col-sm-12 col-md-10">
```
Comment (Orange-walker-95) [inline app/views/contracts/index.html.erb]: Indentation

File: app/views/samples/_sample_table.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      <thead>
+        <tr>
+          <th>Código</th>
+          <th>Identificación de la Muestra</th>
+          <th>Tipo de Muestra</th>
+          <th>Cliente</th>
+          <th>Fecha</th>
+          <th>Encargado</th>
+          <th>Acciones</th>
+        </tr>
```
Comment (HerbertVidela) [inline app/views/samples/_sample_table.html.erb]: The table body it's missing.

File: app/views/sample_methods/_sample_methods_table.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<div class="content">
+  <div class="material-datatables">
+    <table id="sample-methods-table" class="table table-no-bordered table-hover" >
+      <thead>
+        <tr>
+          <th>Código</th>
+          <th>Identificación de la Muestra</th>
+          <th>Acciones</th>
+        </tr>
```
Comment (HerbertVidela) [inline app/views/sample_methods/_sample_methods_table.html.erb]: The table body it's missing.

File: app/views/quotations/_quotations_table.html.erb [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    <table id="quotation-table" class="table table-no-bordered table-hover" >
+      <thead>
+        <tr>
+          <th>Código</th>
+          <th>Código de Solicitud</th>
+          <th>Laboratorio</th>
+          <th>Fecha de Recepción</th>
+          <th>Estado</th>
+          <th>Acciones</th>
+        </tr>
```
Comment (HerbertVidela) [inline app/views/quotations/_quotations_table.html.erb]: The table body it's missing.

File: app/views/contracts/_contracts_table.html.erb [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    <table id="contract-table" class="table table-no-bordered table-hover" >
+      <thead>
+        <tr>
+          <th>Código</th>
+          <th>Código de Cotización</th>
+          <th>Laboratorio</th>
+          <th>Fecha de Registro</th>
+          <th>Estado</th>
+          <th>Acciones</th>
+        </tr>
```
Comment (HerbertVidela) [inline app/views/contracts/_contracts_table.html.erb]: The table body it's missing

File: app/views/samples/_sample_table.html.erb [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+      </thead>
```
Comment (Orange-walker-95) [inline app/views/samples/_sample_table.html.erb:14]: The table body it's missing.

File: app/views/sample_methods/_sample_methods_table.html.erb [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+      </thead>
+    </table>
```
Comment (Orange-walker-95) [inline app/views/sample_methods/_sample_methods_table.html.erb]: The table body it's missing.

File: app/views/quotations/_quotations_table.html.erb [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+      </thead>
+    </table>
```
Comment (Orange-walker-95) [inline app/views/quotations/_quotations_table.html.erb]: The table body it's missing.

File: app/views/contracts/_contracts_table.html.erb [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+      </thead>
```
Comment (Orange-walker-95) [inline app/views/contracts/_contracts_table.html.erb:13]: The table body it's missing.

**Pull request merged**